libc/test/integration/src/pthread/pthread_create_test.cpp

```diff
@@ -47,7 +47,7 @@ static void *successThread(void *Arg) {
   pthread_t th = LIBC_NAMESPACE::pthread_self();
   auto *thread = reinterpret_cast<LIBC_NAMESPACE::Thread *>(&th);
 
-  ASSERT_EQ(libc_errno, 0);
+  ASSERT_EQ(static_cast<int>(libc_errno), 0);
```

Comment:
I think this deserves an `ASSERT_ERREQ` macro in our gtest.
The `static_cast<int>` adds a lot of clutter and will be surprising to newcomers.

Given the size of this change I'd like to do it as a separate change if possible. I'll come back with such a patch and ping here.

Reply:
https://github.com/llvm/llvm-project/pull/79573

Reply:
Thanks for cleaning this up!  It will make this patch much cleaner.

Reply:
I'll merge with #79573 and force push this so that the `libc_errno` commit will be much cleaner for reviewing.